playbooks/roles/privoxy/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Install Privoxy
+  apt:
+    name: privoxy
```

Comment:
Is privoxy necessary for the scope of the changes?

Reply:
Oops. I was tinkering with this, and seems I crossed the streams.